ci/prow/Dockerfile.fcos

```diff
@@ -7,4 +7,4 @@ RUN ./ci/build.sh && make install DESTDIR=$(pwd)/install && tar -C install -czf
 
 FROM quay.io/fedora/fedora-coreos:testing-devel
 COPY --from=builder /srv/install.tar /tmp
-RUN tar -xvf /tmp/install.tar && ostree container commit 
+RUN tar -C / -xvf /tmp/install.tar && ostree container commit
```

Comment:
Was this really broken though?  Won't the current working directory default to `/` here?

I agree this logic is clearer and more robust though.

Reply:
> Won't the current working directory default to `/` here?

Yes, you are correct, the default is `/` here, maybe no need for the change. Sorry for the silly noise, close this now.